dts/arm/broadcom/valkyrie.dtsi

```diff
@@ -0,0 +1,58 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright 2018 Broadcom.
+ */
+
+#include <arm/armv7-m.dtsi>
+#include <broadcom/valkyrie-irq.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m7";
+			reg = <0>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			mpu: mpu@e000ed90 {
+				compatible = "arm,armv7m-mpu";
+				reg = <0xe000ed90 0x40>;
+				arm,num-mpu-regions = <16>;
+			};
+		};
+	};
+
+	sram0: memory@400000 {
```

Comment:
I assume the sram is part of the SoC?  If so can you move this node inside the SoC container?

Reply:
> I assume the sram is part of the SoC? If so can you move this node inside the SoC container?

Yes, sram is internal to the SoC.  I will move this node instead the SoC container.